### PR TITLE
fix for issue #10086

### PR DESCRIPTION
**Pull request by Jiurong-Yang**
A simple fix for the timeout issue for appium-android-driver when installing app. You can reference the issue number for more information. Just trying to get one that passes the CI test.

### CODE REVIEW

**Comment by jlipps** [comment]
@Jiurong-Yang the restriction is there precisely to _keep_ Appium open source, to prevent anyone's employer from claiming that code they submitted to Appium is actually copyrighted. It's a pain, but it's good for the project.

File: lib/android-helpers.js [hunk 1]
```diff
@@ -273,7 +273,9 @@ helpers.reinstallRemoteApk = async function (adb, localApkPath, pkg,
 
 helpers.installApkRemotely = async function (adb, opts) {
   let {app, appPackage, fastReset, androidInstallTimeout} = opts;
-
+  if (androidInstallTimeout ==null) {
```
Comment (jlipps) [inline lib/android-helpers.js]: it's better to say `_.isNull(androidInstallTimeout)`, but that may not do exactly what we want. Probably more like if it's null or undefined.

Comment (Jiurong-Yang) [reply lib/android-helpers.js]: Can you elaborate on this? Does .isNull and == null have different behavior? To be honestly I've never written a line of Java in my life so I'm not very familiar to all the syntax and language specific nuances. So I'm always open to suggestions

**Comment by mykola-mokhnach** [comment]
I'd rather close it in favour of https://github.com/appium/appium-android-driver/pull/319, since this part of code is completely reworked there

**Comment by Jiurong-Yang** [comment]
Closing because another contributor is already working on it.

**Pull request closed without merging**